### PR TITLE
Issue 19: Use bincount instead of unique

### PR DESCRIPTION
**Pull request by xulaus**
This solves issue #19, resulting in expected speed boosts and lower version numpy dependency.

### CODE REVIEW

File: CHAID/CHAID.py [hunk 1]
```diff
@@ -358,16 +357,23 @@ def generate_best_split(self, ind, dep):
         """ internal method to generate the best split """
         split = CHAIDSplit(None, None, None, 1)
         relative_split_threshold = 1 - self.split_threshold
+        # We add one to the dependent array as when an answer hasn't been given
```
Comment (Rambatino) [inline CHAID/CHAID.py]: we commenting now?

File: CHAID/__main__.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def main():
         predictions.name = 'node_id'
         data = pd.concat([data, predictions], axis=1)
         print(data.to_csv())
-    elif nspace.predictions:
+    elif nspace.predict:
```
Comment (Rambatino) [inline CHAID/__main__.py]: what?

File: CHAID/CHAID.py (hunk 1 above)
Comment (xulaus) [inline CHAID/CHAID.py]: Yeah, well. I think in cases like this where you can come back and be like "Why +1" its best to have some indication of why, either a comment or a well named constant. Here a comment seemed to make most sense

File: CHAID/__main__.py (hunk 2 above)
Comment (xulaus) [inline CHAID/__main__.py]: Silly bug on the mainline. I've fixed it in master (without review soz lol), so a rebase will make this change go away.

File: CHAID/CHAID.py (hunk 1 above)
Comment (Rambatino) [inline CHAID/CHAID.py]: Agreed

**Comment by Rambatino** [comment]
@xulaus when you're back we'll get this in.
Seems like some solid work. We'll rebase it onto master and then update the new code too.

**Comment by Rambatino** [comment]
Can you incorporate this into the ordinal variables branch?

**Comment by xulaus** [comment]
This is way old, I'm killing it

**Pull request closed without merging**